### PR TITLE
Minor: Update exected output due to logical conflict

### PR DESCRIPTION
**Pull request by alamb**
## Which issue does this PR close?
N/A

## Rationale for this change

CI is failing on main, for example https://github.com/apache/datafusion/actions/runs/10251066368/job/28358425606

I think it is a logical conflict between https://github.com/apache/datafusion/pull/11795 and https://github.com/apache/datafusion/pull/11814

## What changes are included in this PR?
Update test output

## Are there any user-facing changes?

No, this is just a development process change

### CODE REVIEW

File: datafusion/sqllogictest/test_files/aggregate.slt [hunk 1]
```diff
@@ -4530,21 +4530,21 @@ EXPLAIN SELECT DISTINCT c3, min(c1) FROM aggregate_test_100 group by c3 limit 5;
 ----
 logical_plan
 01)Limit: skip=0, fetch=5
-02)--Aggregate: groupBy=[[aggregate_test_100.c3, MIN(aggregate_test_100.c1)]], aggr=[[]]
-03)----Aggregate: groupBy=[[aggregate_test_100.c3]], aggr=[[MIN(aggregate_test_100.c1)]]
+02)--Aggregate: groupBy=[[aggregate_test_100.c3, min(aggregate_test_100.c1)]], aggr=[[]]
```
Comment (alamb) [inline datafusion/sqllogictest/test_files/aggregate.slt:4533]: the case of the aggregate name is different

**Review by lewiszlw — APPROVED** [review]
I noticed this too and planning to fix it.

**Comment by alamb** [comment]
✅

**Comment by alamb** [comment]
Thanks for the speedy review @lewiszlw

**Pull request merged**